### ISSUE TITLE
onSemanticTokens e.startsWith is not a function error from vscode 'ansible server' output

### ISSUE THREAD

**Issue by piengeng** [ansible/ansible-lint#2614]
```
[Error - 9:48:54 am] An error occurred in 'onSemanticTokens' handler: [TypeError] e.startsWith is not a function
TypeError: e.startsWith is not a function
    at t.isTaskKeyword (/home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:102619)
    at /home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:124711
    at Array.filter (<anonymous>)
    at /home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:124676
    at Generator.next (<anonymous>)
    at /home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:120967
    at new Promise (<anonymous>)
    at r (/home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:120712)
    at b (/home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:124568)
    at /home/henry/.vscode/extensions/redhat.ansible-1.0.90/out/server/src/server.js:2:40175
```

**Comment by ssbarnea** [comment]
Not sure why this issue was reported on linter when it is obvious that it does not come from its code.
